hail/src/main/scala/is/hail/types/TypeWithRequiredness.scala

```diff
@@ -339,13 +339,17 @@ case class RTable(rowFields: Seq[(String, TypeWithRequiredness)], globalFields:
 
   def unionKeys(req: RStruct): Unit = key.foreach { n => field(n).unionFrom(req.field(n)) }
   def unionKeys(req: RTable): Unit = {
+    if (key.length > req.key.length)
+      println()
```

Comment:
Why this `println`?

Reply:
Sorry, leftover from debugging. The println is just a dummy instruction to stick a breakpoint on.